test.html

```diff
@@ -0,0 +1,26 @@
+<script type="importmap">
+{
+  "imports": {
+    "eventemitter3": ["https://unpkg.com/@byungi/event-emitter/dist/index.esm.js"],
```

Comment:
This alias is functionally equivalent to eventemitter3 and enables the test to run, as eventemitter3 itself does not export esm.